hbase-common/src/test/java/org/apache/hadoop/hbase/io/compress/TestCodecPool.java

```diff
@@ -41,13 +44,21 @@
 import org.apache.hadoop.io.compress.zlib.BuiltInGzipDecompressor;
 import org.junit.Before;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 
 /**
  * Along with CodecPool, this is copied from the class of the same name in hadoop-common. Modified
  * to accommodate changes to HBase's CodecPool.
  */
+@Category({ MiscTests.class, SmallTests.class })
```

Comment:
Ah, I bet this is what did it. Without a Category annotation, the plugin didn't have instructions as to whether it should run your test, so it didn't.

Still, i vaguely recall the build would fail when a test was insufficiently decorated.